tests/rkt_auth_test.go

```diff
@@ -0,0 +1,232 @@
+// Copyright 2015 CoreOS, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"strings"
+	"testing"
+
+	"github.com/coreos/rkt/Godeps/_workspace/src/github.com/ThomasRooney/gexpect"
+	taas "github.com/coreos/rkt/Godeps/_workspace/src/github.com/endocode/test-aci-auth-server/lib"
+	"github.com/coreos/rkt/rkt/config"
+)
+
+func TestAuthSanity(t *testing.T) {
+	skipDestructive(t)
+	removeDataDir(t)
+	server := runServer(t, taas.None)
+	defer server.Close()
+	successfulRunRkt(t, server.URL, "none1")
```

Comment:
> I think there should be a comment somewhere saying that "none1" is a directory and that the test server accepts any directory.

The parameter name of the succesfulRunRkt already says that it is a directory, no? But yeah, the comment about any directory should be somewhere.

> It is a bit confusing that server.URL is not the full URL but just the scheme and host.

Uh, "just the scheme and host" _is_ a URL. In this case, `server.URL` is a URL of the server.

> The parameters host and dir are passed together in all the functions. It might be clearer if it was:
> ```
> successfulRunRkt(t, server.URL+"/none1/")
> ```
> Then it would be obvious to the reader what it is?

I don't think so. It would be clearer to the reader when it were
```
successfulRunRkt(t, server.URL+"/none1/prog.aci")
```
With what is currently and what you propose reader will have to go and see what this runRkt business is all about. Or we repeat that `prog.aci` everywhere.